realm/realm-library/build.gradle

```diff
@@ -74,7 +74,8 @@ android {
 
     buildTypes {
         debug {
-            testCoverageEnabled = true
+            // FIXME: If enabled, crashes with https://issuetracker.google.com/issues/37116868
+            testCoverageEnabled = false
```

Comment:
when it is enabled, debugger cannot show local vars which is another bug ... so disable this is 👍